selfdrive/modeld/dmonitoringmodeld.py

```diff
@@ -132,19 +129,26 @@ def main():
   pm = PubMaster(["driverStateV2"])
 
   calib = np.zeros(CALIB_LEN, dtype=np.float32)
+  model_transform = np.zeros((3, 3), dtype=np.float32)
+  transform_set = False
   # last = 0
 
   while True:
     buf = vipc_client.recv()
     if buf is None:
       continue
 
+    if not transform_set:
+      from_intr = _os_fisheye.intrinsics if buf.width > 2000 else _ar_ox_fisheye.intrinsics
```

Comment:
use `DEVICE_CAMERAS` from `common.transformations.camera` to lookup the intrinsics